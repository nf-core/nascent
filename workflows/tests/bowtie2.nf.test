nextflow_pipeline {

    name "Test Workflow main.nf"
    script "../../main.nf"
    tag "aligner"
    tag "bowtie2"

    test("Should run with bowtie2 (default)") {

        when {
            params {
                outdir = "$outputDir"
                aligner = "bowtie2"
            }
        }

        then {
            assert workflow.success
            assert snapshot(
                workflow.trace.tasks().size(),
                path("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml"),
            ).match()
        }
    }
}
