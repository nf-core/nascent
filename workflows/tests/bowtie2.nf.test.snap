{
    "Should run with bowtie2 (default)": {
        "content": [
            132
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.3"
        },
        "timestamp": "2024-07-20T11:38:39.368341091"
    }
}